data/en/test_schema_context.json

```diff
@@ -48,21 +48,93 @@
                     "id": "preview",
                     "title": "",
                     "content": [{
-                        "title": "period_str: {{ metadata['period_str'] }}"
+                        "description": {
```

Comment:
I don't think the name of this file makes sense, this looks like it's just testing piping of metadata.